### ISSUE TITLE
Error running climate update

### ISSUE THREAD

**Issue by pieterprovoost** [iobis/edna-dashboard#116]
```
Error in `duckdb_result()`:
! rapi_execute: Failed to run query
Error: IO Error: Failed to install 'httpfs'
The file was built for DuckDB version 'v1.1.3', but we can only load extensions built for DuckDB version '19864453f7'.
Backtrace:
     ▆
  1. ├─global get_temperature_species(force = TRUE)
  2. │ └─rfishbase::species(species_list$scientificName)
  3. │   ├─fb_tbl(endpt, server, version) %>% fix_ids()
  4. │   └─rfishbase::fb_tbl(endpt, server, version)
  5. │     └─duckdbfs::open_dataset(urls[tbl])
  6. │       └─duckdbfs:::parse_uri(sources, conn = conn, recursive = recursive)
  7. │         └─duckdbfs:::load_httpfs(conn)
  8. │           ├─DBI::dbExecute(conn, "INSTALL 'httpfs';")
  9. │           └─DBI::dbExecute(conn, "INSTALL 'httpfs';")
 10. │             ├─DBI::dbSendStatement(conn, statement, ...)
 11. │             └─DBI::dbSendStatement(conn, statement, ...)
 12. │               ├─DBI::dbSendQuery(conn, statement, ...)
 13. │               └─duckdb::dbSendQuery(conn, statement, ...)
 14. │                 └─duckdb (local) .local(conn, statement, ...)
 15. │                   └─duckdb:::duckdb_result(connection = conn, stmt_lst = stmt_lst, arrow = arrow)
 16. │                     └─duckdb:::duckdb_execute(res)
 17. │                       └─duckdb:::rethrow_rapi_execute(...)
 18. │                         ├─rlang::try_fetch(...)
 19. │                         │ ├─base::tryCatch(...)
 20. │                         │ │ └─base (local) tryCatchList(expr, classes, parentenv, handlers)
 21. │                         │ │   └─base (local) tryCatchOne(expr, names, parentenv, handlers[[1L]])
 22. │                         │ │     └─base (local) doTryCatch(return(expr), name, parentenv, handler)
 23. │                         │ └─base::withCallingHandlers(...)
 24. │                         └─duckdb:::rapi_execute(stmt, arrow, integer64)
 25. ├─rfishbase:::fix_ids(.)
 26. │ ├─"Speccode" %in% colnames(full_data)
 27. │ └─base::colnames(full_data)
 28. │   └─base::is.data.frame(x)
 29. └─base::.handleSimpleError(...)
 30.   └─rlang (local) h(simpleError(msg, call))
 31.     └─handlers[[1L]](cnd)
 32.       └─duckdb:::rethrow_error_from_rapi(e, call)
 33.         └─rlang::abort(msg, call = call)
Execution halted
```

**Comment by silasprincipe** [comment]
Something with DuckDB version. I will need to check what is happening

**Comment by pieterprovoost** [comment]
Related issues:

- https://github.com/duckdb/duckdb-r/issues/623
- https://github.com/duckdb/duckdb-r/issues/600

**Comment by wardappeltans** [comment]
is that the reason why you cannot filter on red list species in the climate tab? e.g. the coral species Pseudodiploria strigosa is CR but in Belize no CR species shows up in the Climate tab. @silasprincipe @pieterprovoost

**Comment by silasprincipe** [comment]
Yes, the new version will solve that.

**Comment by pieterprovoost** [comment]
Fixed by forcing duckdb-r v1.1.0. https://github.com/iobis/shiny/commit/6721b38b12b1ddb2981500d73457826f720256b6
